misc/eni-max-pods.txt

```diff
@@ -828,6 +843,7 @@ t4g.xlarge 58
 trn1.2xlarge 58
 trn1.32xlarge 247
 trn1n.32xlarge 247
+trn2.48xlarge 100
```

Comment:
<img width="974" alt="Screenshot 2024-12-08 at 10 33 10 AM" src="https://github.com/user-attachments/assets/60d121e4-25c5-4788-9b22-4e0474d241b1">

@Pavani-Panakanti  - even this..